src/components/Switchers/index.js

```diff
@@ -34,7 +34,7 @@ class Switcher extends Component {
   }
 
   render() {
-    const { rightOption, leftOption, onClick } = this.props
+    const { rightOption, leftOption } = this.props
```

Comment:
If you don't want inline though you can just run `yarn verify` occasionally in each project